client/src/components/Reservation.jsx

```diff
@@ -1,24 +1,42 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import SetName from './SetName';
 import SearchParams from './SearchParams';
 import TimeSlotSelector from './TimeSlotSelector';
 
+import helper from '../lib/helper';
+
 class Reservation extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
       // timeSlotsLeft: 0,
-      // bookingsMadeToday: 0,
-      // availabilityInfo: [],
+      bookingsMadeToday: 0,
+      availabilityInfo: [],
       // date: '',
       // time: 17,
       // name: '',
       // party: 1,
     };
   }
 
+  componentDidMount() {
+    this.fetch();
+  }
+
+  fetch(date = (new Date()).toISOString().slice(0, 10)) {
+    helper.getReservationInfo(this.props.id, date, (err, data) => {
+      console.log('getReservationInfo callback', this.props.id, data);
+      console.log(this.state.bookingsMadeToday, this.state.availabilityInfo);
```

Comment:
Is your linter ignoring these files? Just asking considering the use of console.log

Reply:
Yeah without the console.log my linter will give errors about unused state variables.  So the console log is there to use the state variables because console.log is only a warning and not an error.